### ISSUE TITLE
New WineD3D requires new functions implementations

### ISSUE THREAD

**Issue by Svyatpro** [tumagonx/XomPie#5]
Is it possible to implement these new functions?

gdi32:
D3DKMTCreateDevice
D3DKMTDestroyDevice
D3DKMTOpenAdapterFromGdiDisplayName
D3DKMTSetVidPnSourceOwner

As I researched the most important to function are:
D3DKMTCreateDevice
D3DKMTOpenAdapterFromGdiDisplayName

They are implemented in Wine with these two patches.
https://github.com/wine-mirror/wine/commit/733a92df4bdeb1f4592155ee6fb4d5a7542113bf#diff-b333ec2bd91f466dbe68225f9cc09a4f

https://github.com/wine-mirror/wine/commit/d37030528eef36014b51171fecc9c381b67cb11b#diff-b333ec2bd91f466dbe68225f9cc09a4f

**Comment by tumagonx** [comment]
I don't know even make d3d9ex works here, haven't catching up with KMT, I thought you're ahead of me in direct3d stuff!?

**Comment by Svyatpro** [comment]
yes but not in KMT stuff. Personally, I don't develop One-Core-API, I just have some fun work with WineD3D on Windows. Now starting from WineD3D 4.19 it requires some more D3DKMT* functions without which I can't make it work. It is now very hard to revert those KMT pathces.
